src/v/kafka/server/handlers/fetch.cc

```diff
@@ -707,6 +707,25 @@ ss::future<response_ptr> op_context::send_response() && {
     final_response.data.error_code = response.data.error_code;
     final_response.data.session_id = response.data.session_id;
 
+    /// Account for internal topic bytes for usage
+    const auto& internal_topics
+      = config::shard_local_cfg().kafka_nodelete_topics();
```

Comment:
@dcoded

Reply:
So maybe we should make an exemption for `__consumer_offsets`?

Reply:
>Can you think of an access pattern that might be problematic if we don't change for it

One where a customer reads that topics enough times that it moves the needle. I doubt that exists, fwiw. The flip side of this is: if it isn't going to move the needle, then why special case it?

Reply:
To expand on this, redpanda will not go through the kafka API when publishing / consuming from `__consumer_offsets` partitions. The source here only accounts from bytes via fetch requests.

Reply:
Let's omit it. That way there's no discrepancy of billing between SR/PP and the Kafka API.

Reply:
Apologies. Can you help educate me on other internal topics? If they are for Redpanda to operate/support/debug Cloud then we should not bill customers for it.

Reply:
agree with the definition provided by @dcoded 

* so no metering for audit and e2e probe
* i would have guessed that we want to meter kafka client fetches to consumer offsets, but i'm still unclear what the consensus opinion is here. is everyone on the same page about what the scenarios are?
* _schemas it sounds like we aren't metering because we want this to be hidden behind schema registry

Reply:
Ok sounds good i will be hardcoding this list